dnp3/src/master/request.rs

```diff
@@ -407,72 +379,293 @@ impl CommandHeaders {
     }
 }
 
+#[derive(Clone)]
```

Comment:
I guess it is cloned in the FFI, to make a header set reusable?

Reply:
Cloned b/c you can't take ownership of a reference. In the interests of time I'm going to move onto another feature. We can revisit the macro/generic code if desired at some point.